.gitignore

```diff
@@ -17,3 +17,6 @@
 
 # Ignore application configuration
 /config/application.yml
+/coverage/
+spec/examples.txt
+.byebug_history
```

Comment:
These are already on master, and would need to be explicitly deleted to actually get rid of them. I created https://trello.com/c/w65tztjO, and recommend that we leave out the deletion of byebug and the addition of the gitignores here in order to avoid conflicts.

That said, I would love it if you tackled that ticket 😄